### ISSUE TITLE
Forward and backward search not working in Windows 10

### ISSUE THREAD

**Issue by t-morales** [texstudio-org/texstudio#205]
Hi!

For some reason forward and backward search no longer works. It used to work but now ctrl+click or "go to source" does nothing. The same when I click in preview, the pdf just opens on the first page. 
The option -synctex=1 is given and the synctex file is generated. I tried reinstall and deleting personal profile but the problem persists. Any help will be much appreciated.

### Environment
- TeXstudio: TeXstudio 2.12.6
- Qt:  5.9.1,
- OS: Windows 10
- TeX distribution: Texlive with WSL

**Comment by sunderme** [comment]
we need a test case: tex-file, synctex.gz and pdf.

**Comment by sunderme** [comment]
furthermore try the latest version of txs (2.12.8)

**Comment by t-morales** [comment]
Just any simple example does not work for me. I am attaching one.
I already installed the new version 2.12.8 and used the portable version 2.12.8 with the same results.
At least the previous version 2.12.6 used to work but it stopped working no idea why. My guess is that something changed in the configuration at some point (in texstudio or windows) but no idea what or how to debug it.
[test.zip](https://github.com/texstudio-org/texstudio/files/2146838/test.zip)

**Comment by sunderme** [comment]
from the log I gather that the document is on a mounted (ntfs?) drive.
Can you try to compile and use it from your home/sub under home ?

**Comment by sunderme** [comment]
okay, I see that you are using WSL. Is latex running on wsl and txs on windows ?

**Comment by t-morales** [comment]
Yes, latex is in WSL and txs on windows. It's the same configuration that I've using for quite some time.
